cmd/test-utils_test.go

```diff
@@ -2361,3 +2361,25 @@ func getEndpointsLocalAddr(endpoints EndpointList) string {
 
 	return globalMinioHost + ":" + globalMinioPort
 }
+
+// fetches a random number between range min-maz
+func getRandomRange(min, max int) int {
+	rand.Seed(time.Now().Unix())
```

Comment:
Looks like my earlier point about seeding the RNG was not understood. If you pick a seed like this, the value of this test is low - the seed changes based on time and each time this code runs it will pick a new seed. This means that each time the generated random number is different and even if there is a bug in the code that you are testing, and it is found via this test, it will not be reproducible, because in the next run the test failure may disappear.

Using random numbers in testing should not be taken lightly. If you must use a randomization based approach, it needs to be done properly. Here the seed value needs to be picked in the calling code only once - that sets the sequence of random numbers that will be generated in all further randomization steps. So it would work like:

1. Seed RNG.
2. Select length of string randomly (**without resetting the seed!**)
3. Randomly permute the byte-array (**without resetting the seed!**)
4. Verify results of the code being tested.